rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/250_moving_fn.yml

```diff
@@ -6,6 +6,10 @@ setup:
 ---
 "Bad window":
 
+  - skip:
+      version: " - 7.99.0"
+      reason:  "calendar_interval added in 7.1"
```

Comment:
Should we have 2 versions of this test, one for the legacy interval and one for the new intervals?

Reply:
This test already catches a warning (the bad window), and I don't think the yaml runner allows catching two warnings at the same time.  We'd also need to catch the deprecation warning for the legacy `interval`

It's a good question in general though.  I tried to test both versions in tests about the date_histo, but converted to the "correct" new version for all other tests.  If we want we can add back some of the legacy intervals in some places, or all places.

Reply:
I misread, the existing catch is for an exception.  I'll add a block for warnings and duplicate the test.